netperf_tool/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>netperf_tool</name>
+  <version>0.19.0</version>
```

Comment:
Why 0.19.0?

Reply:
IIUC, we have to use the same version number for all packages in the same repo.
And currently we're on [0.19.0](https://github.com/ros2/ros2cli/blob/master/ros2doctor/package.xml).

Maybe I can just leave it as `0.0.0` and it will be fixed in the next `catkin_prepare_release`, I'm not sure how it works.

Reply:
The version number has to match all others in this repository, otherwise we won't be able to do a release.

Obviously this doesn't apply if we decide to put it in a different repository.

Reply:
Oh right, my bad.